charts/spire/charts/spire-server/values.yaml

```diff
@@ -87,6 +87,12 @@ trustDomain: example.org
 
 bundleConfigMap: spire-server
 
+federation:
+  bundleEndpoint:
+    enabled: false
```

Comment:
Should we lift this one level up?

So federation.enabled?

Reply:
Good question.... what else might be enabled if federation is enabled?